ding2.profile

```diff
@@ -829,3 +829,24 @@ function ding2_set_cookie_page() {
   // display EU Cookie Compliance popup: anonymous user, authenticated user
   // administer EU Cookie Compliance popup: administrators, local administrator
 }
+
+/**
+ * Enabling Shortcuts plugin for Administration Menu module.
+ */
+function ding2_admin_menu_shortcuts() {
+  if (module_exists('admin_menu')) {
+    $content = variable_get('admin_menu_components', array());
+
+    if (empty($content)) {
+      module_load_include('inc', 'admin_menu', 'admin_menu');
+      admin_menu_theme_settings();
```

Comment:
Why is calling `admin_menu_theme_settings()` necessary here?

The method is used to generate a form, the return value is not used and [the function does not seem to have any sideeffects](http://api.drupalhelp.net/api/admin_menu/admin_menu.inc/function/admin_menu_theme_settings/7.3).

If there is a reason please add a comment explaining why.